package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-cognito-forms",
-  "version": "0.28.0",
+  "version": "0.29.0",
```

Comment:
Please don't increment the version number, I'll do that as part of an automated process when I publish the new one.